vxquery-core/src/main/java/org/apache/vxquery/xmlquery/translator/XMLQueryTranslator.java

```diff
@@ -1540,6 +1542,38 @@ private LogicalVariable translatePathExpr(PathExprNode pe, TranslationContext tC
                     FilterExprNode filterNode = (FilterExprNode) pathNode;
                     predicates = filterNode.getPredicates();
                     ctxExpr = vre(translateExpression(filterNode.getExpr(), tCtx));
+                } else if (ASTTag.POSTFIX_EXPRESSION.equals(pathNode.getTag())) {
+                    PostfixExprNode postfixNode = (PostfixExprNode) pathNode;
+                    List<ASTNode> args = postfixNode.getArgs();
+                    ILogicalExpression expr = vre(translateExpression(postfixNode.getExpr(), tCtx));
+                    List<ILogicalExpression> arguments = new ArrayList<ILogicalExpression>();
+                    if (args != null && !args.isEmpty()) {
+                        for (ASTNode an : args) {
+                            if (an.getTag() == ASTTag.ARGUMENT_LIST) {
+                                ArgumentListNode argNode = (ArgumentListNode) an;
+                                for (ASTNode en : argNode.getExpression()) {
+                                    ILogicalExpression argument = vre(translateExpression(en, tCtx));
+                                    arguments.add(argument);
+                                    if (args.size() == 1) {
+                                        ctxExpr = sfce(BuiltinOperators.VALUE, expr, argument);
+                                    } else {
+                                        if (an == args.get(0))
```

Comment:
I'm not sure that I understand the position logic here?
Could you add a comment that explains?